docs/tutorial/get-started.rst

```diff
@@ -165,44 +166,15 @@ The output should include user details as shown below, with auto-generated acces
     "max_buckets": 1000,
     "subusers": [],
     "keys": [
-        {
-            "user": "user",
-            "access_key": "NJ7YZ3LYI45M4Q1A08OS",
-            "secret_key": "H7OTclVbZIwhd2o0NLPu0D7Ass8ouSKmtSewuYwK",
-            "active": true,
-            "create_date": "2024-11-28T13:07:41.561437Z"
-        }
-    ],
-    ...
-
-Set user secrets
-~~~~~~~~~~~~~~~~
-Let's define secrets for this user, setting ``access_key`` to ``foo``, and ``--secret-key`` to ``bar``.
-
-.. code-block:: none
-
-    sudo radosgw-admin key create --uid=user --key-type=s3 --access-key=foo --secret-key=bar
-
-.. terminal::
-
-    ... 
-    [
-        {
-            "user": "user",
-            "access_key": "NJ7YZ3LYI45M4Q1A08OS",
-            "secret_key": "H7OTclVbZIwhd2o0NLPu0D7Ass8ouSKmtSewuYwK",
-            "active": true,
-            "create_date": "2024-11-28T13:07:41.561437Z"
-        },
         {
             "user": "user",
             "access_key": "foo",
             "secret_key": "bar",
             "active": true,
-            "create_date": "2024-11-28T13:54:36.065214Z"
+            "create_date": "2024-11-28T13:07:41.561437Z"
         }
     ],
-   ...
+    ...
 
 Consuming the storage
```

Comment:
Please add some content here before the second heading. Headings require content and should not be followed directly by a subheading.